guest/api/types.go

```diff
@@ -57,6 +59,12 @@ type PreFilterPlugin interface {
 	PreFilter(state CycleState, pod proto.Pod) (nodeNames []string, status *Status)
 }
 
+// PreFilterExtension is a WebAssembly implementation of framework.PrefFilterExtensions.
```

Comment:
```suggestion
// PreFilterExtensions is a WebAssembly implementation of framework.PrefFilterExtensions.
```

Reply:
seems silly, but this is to make sure godoc picks it up

Reply:
Thanks!